src/main/java/com/google/api/codegen/transformer/nodejs/NodeJSPackageMetadataTransformer.java

```diff
@@ -14,18 +14,23 @@
  */
 package com.google.api.codegen.transformer.nodejs;
 
+import com.google.api.codegen.InterfaceView;
 import com.google.api.codegen.config.ApiConfig;
 import com.google.api.codegen.transformer.ModelToViewTransformer;
-import com.google.api.codegen.viewmodel.PackageMetadataView;
 import com.google.api.codegen.viewmodel.ViewModel;
+import com.google.api.codegen.viewmodel.metadata.IndexRequireView;
+import com.google.api.codegen.viewmodel.metadata.IndexView;
+import com.google.api.codegen.viewmodel.metadata.PackageMetadataView;
+import com.google.api.tools.framework.model.Interface;
 import com.google.api.tools.framework.model.Model;
+import com.google.common.collect.Iterables;
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 
 /** Responsible for producing package metadata related views for NodeJS */
 public class NodeJSPackageMetadataTransformer implements ModelToViewTransformer {
   private static final String PACKAGE_FILE = "nodejs/package.snip";
+  private static final String INDEX_FILE = "nodejs/index.snip";
```

Comment:
Oh okay I see. I thought everything generated by packman was part of the metadata. I can make it a part of gapic codegen.